src/pages/Lend/LendByCategoryPage.vue

```diff
@@ -242,6 +243,20 @@ export default {
 				}
 			}
 
+			// track recommended loans row
+			if (this.showRecommendedLoansRow) {
+				if (this.recommendedLoans !== undefined && this.recommendedLoans.length > 0) {
+					_each(this.recommendedLoans, (loan, loanIndex) => {
+						loanIds.push({
+							r: 95,
```

Comment:
If we can't get the exact row position I would recommend putting in impossible value here, such as the #2 that is suggested -- we should leave 95 just for the loan channel.